### ISSUE TITLE
Porvide a default config

### ISSUE THREAD

**Issue by mthjs** [tcoopman/image-webpack-loader#88]
Just defining the loader in webpack 2 like:
```javascript
module.exports = {
    ...
    module: {
        {
          test: /\.(gif|jpe?g|png|svg)(\?.*)?$/,
          use: [
            'file-loader',
            'image-webpack-loader',
          ],
        },
        {
          test: /\.(eot|ttf|woff2?)(\?.*)?$/,
          loader: 'file-loader',
          options: {
            name: '[name].[ext]',
          },
        },
      ],
    },
    ...
  };
};
```
will cause webpack to fail with the following error `Module build failed: TypeError: Cannot read property 'bypassOnDebug' of null`.

It seems there is no default config available anymore. This could be resolved by falling back on the `getLegacyLoaderConfig` call in `index.js` like :
```javascript
var config = this.version === 2 ?
    loaderUtils.getOptions(this)
    : getLegacyLoaderConfig(this, "imageWebpackLoader");

config = config || getLegacyLoaderConfig(this, "imageWebpackLoader");
```
Of course people should define their config properly; but providing a default config would be nice to help them get started.

**Comment by VitorLuizC** [comment]
A workaround to avoid this error is provide a empty object on webpack.config.

```js
{
  test: /\.(gif|jpe?g|png|svg)(\?.*)?$/,
    use: [
      'file-loader',
      {
        loader: 'image-webpack-loader',
        options: {}
      }
    ]
}
```

Also, defining a default object on config fix this issue.

```js
var config = this.version === 2 ?
    loaderUtils.getOptions(this)
    : getLegacyLoaderConfig(this, "imageWebpackLoader");

config = config || {};
```

**Comment by VitorLuizC** [comment]
I've forked this repo and its already fixed on master by #87. So, until they publish U can fix by installing this module from git
```sh
npm install --save-dev tcoopman/image-webpack-loader
```

**Comment by tcoopman** [comment]
closed by #87
